test/lit.cfg

```diff
@@ -104,6 +104,12 @@ def get_lldb_python_path(lldb_build_root):
         return None
     return subprocess.check_output([lldb_path, "-P"]).rstrip().decode('utf-8')
 
+def get_lldb_python_from_path(lldb_path):
+    for c in lldb_path.split('/'):
```

Comment:
Could you use `os.sep` instead of `/` please?

Reply:
Even better, I used `os.path.split` to separate the path into components (which required rewriting this to a recursive structure instead of iterative).  That should work correctly on all platforms, even those where there are multiple valid path separators.